### PR TITLE
Fixed wind effect issue when changing scene.

### PR DESCRIPTION
**Pull request by ultrabot2022**
https://user-images.githubusercontent.com/80860637/181345121-1b42cfec-2a2d-4523-9e91-feb662b78feb.mp4

This is the video of windy effect issue when changing scene.

### CODE REVIEW

File: avatars/simulation/wind.js [hunk 1]
```diff
@@ -24,6 +24,14 @@ const update = (timestamp, headPosition, springBoneManager) => {
       return -1;
     }
 
+    const _handleNormal = () =>{
+      for (const springBones of springBoneManager.springBoneGroupList) {
+        for (const o of springBones) {
+          o.gravityDir = new THREE.Vector3()
```
Comment (lalalune) [inline avatars/simulation/wind.js:30]: the 'new' allocates memory, can we create a default and use it repeatedly

Comment (ultrabot2022) [reply avatars/simulation/wind.js:30]: Ah, I didn't think of that. I'll update soon. :)

**Comment by lalalune** [comment]
@ultrabot2022 please edit your comment to reference the Github issues this fixes and record a video of it working so QA knows what to look for.

**Comment by ultrabot2022** [comment]
https://user-images.githubusercontent.com/80860637/181345121-1b42cfec-2a2d-4523-9e91-feb662b78feb.mp4

This is the video of windy effect issue when changing scene.

**Review by lalalune — APPROVED** [review]

**Review by lalalune — APPROVED** [review]

**Pull request merged**